### ISSUE TITLE
Can support Motion JPEG

### ISSUE THREAD

**Issue by AmpletecGo** [phoboslab/jsmpeg#219]

**Comment by kosso** [comment]
Yes. eg: If you were using 'mjpeg-streamer' to create your video, you can then use ffmpeg to send it to the server: 

`ffmpeg -i http://YOUR-MJPEG-HOST:8080/?action=stream -f mpegts -codec:v mpeg1video -s 352x288 -b:v 300k -bf 0 http://YOUR-JSMPEG-SERVER:8088/password`
